apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/AppConfigurationTest.java

```diff
@@ -362,7 +426,7 @@ public void testLoadRdaSourceConfig() {
   @Test
   public void testLoadRdaServerJobConfig() {
     var settingsMap = new HashMap<String, String>();
-    var configLoader = ConfigLoader.builder().addSingle(settingsMap::get).build();
+    final var configLoader = AppConfiguration.createConfigLoaderForTesting(settingsMap::get);
```

Comment:
Does this need to be loaded for each test, or could we set it up once for all tests?

Reply:
Every test uses its own config settings.